src/components/App.vue

```diff
@@ -6,7 +6,7 @@
       <RequestHeaders :body="item.requestHeadersFormatted" />
       <Request :body="item.requestBodyFormatted" />
       <Response :body='item.responseBodyFormatted' :raw="item.responseBody"/>
-      <hr class="border-gray-300 border-1 my-2" />
+      <hr class="border-slate-400 border-dashed border-2 my-4" />
```

Comment:
I’m not a fan of the dashed border, but I like the idea of changing it. I was thinking maybe something like this?
```html
<hr class="border-gray-300 border-1 my-4 w-3/4 m-auto" />
```

Reply:
Yep, that thick dashed divider didn't look right. I tried to use your solution, but if we want to keep the boxed design, there are too much borders/lines imo, not very UX friendly. More viable design ideas:

1. We can logically box each request without outline, only with background
![image](https://user-images.githubusercontent.com/28686994/195045761-87aeaf37-d1bd-4b1e-b4a4-1d88a39ea091.png)
2. Do the same, but also add divider
![image](https://user-images.githubusercontent.com/28686994/195045980-4cd02dc2-baa0-4049-ba90-c90467b86fbb.png)

What do you think?

Reply:
btw, I also removed that "slate" color from everywhere, and we are only using now the gray color in different saturations.

Reply:
I think the one with the background and without the `<hr>` line looks good 👍

Reply:
seems to me the shadow is unnecessary, but otherwise this looks awesome 👍